build.sh

```diff
@@ -92,6 +92,9 @@ case ${BUILD_TARGET} in
     ;;
 esac
 
+# tweak sorc/mpas/src/core_init_atmosphere/mpas_init_atm_cases.F to make building work
+sed -i -e "5549 s/.*/call mpas_log_write('Interpolating SOILM000')/" sorc/mpas/src/core_init_atmosphere/mpas_init_atm_cases.F
+
```

Comment:
The reason for this tweak is reported in this issue: https://github.com/NOAA-GSL/MPAS-Model/issues/93
Currently, we don't know what is a good way to move forward with this on the MPAS-Model side.
The standard alone CMake can build MPAS-Model, but RDASApp complains about line 5549 of the 'mpas_init_atm_cases.F".   I don't have time to debug this for the moment. But we cannot wait and need to move forward on using the GSL MPAS for RDASApp.

Reply:
Maybe we should bring this to RRFS developer meeting. It is a weird change.

Reply:
@ShunLiu-NOAA That's a good idea! Hope that will elevate the priorities for this issue. Thanks!